src/components/structures/RoomDirectory.tsx

```diff
@@ -164,6 +164,10 @@ export default class RoomDirectory extends React.Component<IProps, IState> {
         this.getMoreRooms();
     };
 
+    private transformSearchTerm(term: string): string {
+        return term.substring(term.indexOf("#") +2);
```

Comment:
This seems like a rather hacky solution, I feel like there ought to be a better way to do this... Also, it would be best to put this into one place and import it from there, so that you don't repeat the code

Reply:
I am working on importing the function from one place and using regex.